src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -73,8 +85,15 @@ export const ButtonGroup = React.forwardRef<HTMLDivElement, ButtonGroupProps>(
       >
         {!!overflowButton && visibleCount < items.length ? (
           <>
+            {overflowButton && overflowPosition === 'start' && (
+              <div>{overflowButton(visibleCount)}</div>
```

Comment:
Why div is needed?

Reply:
it was forgotten before. If you go to the live version and try to hover/focus on the overflow button, you will see messed up borders